src/python/pants/backend/python/target_types.py

```diff
@@ -241,7 +241,9 @@ class PexPlatformsField(StringSequenceField):
     help = (
         "The platforms the built PEX should be compatible with.\n\nThis defaults to the current "
         "platform, but can be overridden to different platforms. You can give a list of multiple "
-        "platforms to create a multiplatform PEX.\n\nTo use wheels for specific "
+        "platforms to create a multiplatform PEX. A multiplatform PEX file will contain platform "
+        "specific wheels for each 3rd party dependency that has some non-Python "
+        "code that requires compilation on each target platform.\n\nTo use wheels for specific "
```

Comment:
If you take the above recommendation about talking about mentioning built wheels earlier, then perhaps something like this:

> You can give a list of multiple platforms to create a multiplatform PEX. There must be built wheels available for all of the foreign platforms, rather than sdists.